drake/CMakeLists.txt

```diff
@@ -159,14 +159,31 @@ message(STATUS "Writing addpath_drake.m and rmpath_drake.m to ${CMAKE_INSTALL_PR
 file(WRITE ${CMAKE_INSTALL_PREFIX}/matlab/addpath_drake.m
 	   "function addpath_drake()\n"
 	   "  mfiledir = fileparts(which(mfilename));\n"
+     "  jarfiledir = fullfile(mfiledir, '..', 'share', 'java');\n"
```

Comment:
This logic needs to go in the drake/addpath_drake.m instead of this one.  the other one definitely gets called.  this one, only maybe.